### PR TITLE
Fix tooltip initiation + hide

### LINKED ISSUE #14: Tooltip for actions is wrong positioned or not initiated
**Context**
Contao version: 4.9.13
Bundle version: 2.12.2
PHP version: 7.4 (php8 not yet supported ;-))
Chrome: 89.0.4389.114 / Firefox: 87.0

**Description**

Sometimes the tooltip position is in the top left corner:
image

Or it stays fixed at the icon position and wont toggle anymore. Even the tooltips of all other actions are not initiated anymore:
image

Same behavior in Firefox - although there the icon position with `align-self: end;` is interpreted different there!
image

I think there is something like this missing:
https://github.com/menatwork/contao-multicolumnwizard-bundle/blob/e9594b1b2dbe3684100a8059fb2e99eed3ff2964/src/Resources/public/js/multicolumnwizard_be_src.js#L668-L675

### PR DESCRIPTION
**Pull request by rabauss**
This fixes #14.

* [x] Init tooltip for new action links
* [x] Fix position of actions for firefox
* [x] Hide the tooltip of the current link (see: https://github.com/contao/contao/issues/732)
* [x] I'm not sure how you generate the assets from the source node package

### CODE REVIEW

**Comment by koertho** [comment]
Thank @rabauss ! I try to look into it today. Builing the assets should be as easy as `yarn install` and `yarn encore prod`, but I can take that, if you like.

**Comment by koertho** [comment]
Hi @rabauss , just looked into your PR and looks good so far. I just added the css change to the correct file, as this was the generated file ;) I also made a new build and corrected the build script, so it should work now without additional yarn magic (yeah, we tried many things until we found a good build workflow ;) )

**Comment by koertho** [comment]
So, rebuild the asset with prod env :) 

What do you mean with: //TODO Hide the tooltip of the current link (see: contao/contao#732)

**Comment by rabauss** [comment]
Thanks for your review and generation of the files!

Currently the tooltip is not hidden if you click remove or add, so the tooltip sometimes stays fixed positioned after the action. That's why I'm waiting for resonse in the core issue. Because I don't know how to hide a tooltip of a selected link.

File: src/Resources/npm-package/js/contao-multi-column-editor-bundle.js [hunk 1]
```diff
@@ -198,6 +198,10 @@ class MultiColumnEditorBundle {
                         MultiColumnEditorBundle.initChosen(widget);
                         MultiColumnEditorBundle.initSortable(isBackend);
 
+                        MultiColumnEditorBundle.hideTooltip();
+                        setTimeout(function () {MultiColumnEditorBundle.hideTooltip();}, 1000); // hide delayed tooltips
```
Comment (rabauss) [inline src/Resources/npm-package/js/contao-multi-column-editor-bundle.js:202]: It's not a nice solution, but I don't know how else to fix it 😪

Comment (koertho) [reply src/Resources/npm-package/js/contao-multi-column-editor-bundle.js:202]: It worked for me and if there is no other solution, it's ok for me. What do think @Defcon0 ?

Comment (Defcon0) [reply src/Resources/npm-package/js/contao-multi-column-editor-bundle.js:202]: Not nice, but if it works, I'm ok with it. Thanks @rabaus !

Comment (rabauss) [reply src/Resources/npm-package/js/contao-multi-column-editor-bundle.js:202]: It's even better than the current state with never hiding tooltips ;-)
Maybe we will find a better solution while implementing a PR for the related issue in the contao-core 😉

Comment (koertho) [reply src/Resources/npm-package/js/contao-multi-column-editor-bundle.js:202]: Sure, let us know :)

**Review by koertho — APPROVED** [review]
I don't see any showstopper and think we can merge this. Are you agree @Defcon0 ?

**Comment by koertho** [comment]
Thank you very much @rabauss !

**Pull request merged**